README.md

```diff
@@ -66,33 +68,41 @@ func main() {
 
     query := `MATCH (p:person)-[v:visited]->(c:country)
            RETURN p.name, p.age, c.name`
-    rs, _ := graph.Query(query)
 
-    rs.PrettyPrint()
+    // result is a QueryResult struct containing the query's generated records and statistics.
+    result, _ := graph.Query(query)
+
+    // Pretty-print the full result set as a table.
+    result.PrettyPrint()
 
-    // Access individual result record.
-    // As long as there are records to consume.
-    for rs.Next() {
-        // Get current record.
-        r := rs.Record()
+    // Iterate over each individual Record in the result.
+    for result.Next() { // Next returns true until the iterator is depleted.
+        // Get the current Record.
+        r := result.Record()
 
-        p_name := r.GetByIndex(0).(string)
-        p_age := r.GetByIndex(1).(int)
-        c_name := r.GetByIndex(2).(string)
-        fmt.Printf("p_name: %s p_age: %d c_name: %s\n", p_name, p_age, c_name)
+        // Entries in the Record can be accessed by index or key.
+        p_name := r.GetByIndex(0)
+        fmt.Printf("\nName: %s\n", p_name)
+        p_age, _ := r.Get("p.age")
```

Comment:
Note that I've removed the explicit casts in the accessors, as the sample worked properly without them for me!